lib/nerves_hub_ca.ex

```diff
@@ -1,18 +1,42 @@
 defmodule NervesHubCA do
-  @moduledoc """
-  Documentation for NervesHubCA.
+  alias NervesHubCA.CFSSLServer
+
+  @type resp :: {:ok, status_code :: non_neg_integer, binary} | {:error, any}
+
+  @doc """
+  https://github.com/cloudflare/cfssl/blob/master/doc/api/endpoint_init_ca.txt
   """
+  @spec init_ca(Map.t() | binary) :: resp
+  def init_ca(params) do
+    CFSSLServer.request(:post, "init_ca", params)
+    |> resp
+  end
 
   @doc """
-  Hello world.
+  https://github.com/cloudflare/cfssl/blob/master/doc/api/endpoint_certinfo.txt
+  """
+  @spec certinfo(Map.t() | binary) :: resp
+  def certinfo(params) do
+    CFSSLServer.request(:post, "certinfo", params)
+    |> resp
+  end
 
-  ## Examples
+  @doc """
+  https://github.com/cloudflare/cfssl/blob/master/doc/api/endpoint_newcert.txt
+  """
+  @spec newcert(Map.t() | binary) :: resp
+  def newcert(params) do
```

Comment:
Since we'll have more than Nerves Hub creating device certs in the long run, what do you think about adding the higher level calls here so that their logic doesn't get lost in the `nerves_hub` codebase? For example, how about adding a `create_device_certificate` call that takes a device's serial number, creates a UUID, and maps the metadata to the appropriate certificate fields?